docs/src/user-guide/guides-retention.md

```
@@ -0,0 +1,167 @@
+  is a local timestamp, then a garbage collection job operating at 16:30 local time will convert
+  `16:30 EDT` to `20:30 UTC`, and the expiry calculation will be `20:30 - 16:00 > 1:00`. In this
+  case, the archive would be considered expired, and would be deleted, even though it wouldn't have
+  actually reached its intended retention period.
+
+  To avoid this issue, either generate logs with UTC timestamps or adjust the retention period to
+  account for the offset:
+  
+  `adjusted_retention_period = retention_period - signed_UTC_offset`
+  :::
+
+- **Search Result Expiry:** 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)